tsconfig.base.json

```diff
@@ -41,6 +41,12 @@
       "@ama-sdk/schematics": [
         "packages/@ama-sdk/schematics"
       ],
+      "@ama-sdk/showcase-sdk": [
+        "packages/@ama-sdk/showcase-sdk/src"
+      ],
+      "@ama-sdk/showcase-sdk/fixtures/jest": [
```

Comment:
Shouldn't we put `"@ama-sdk/showcase-sdk/fixtures"` to not have to refactor test files if one day we move to another framework ?

Reply:
done